client/packages/requisitions/src/ResponseRequisition/DetailView/IndicatorEdit/ListIndicators.tsx

```diff
@@ -29,9 +31,9 @@ export const ListIndicatorLines = ({
         });
       }}
       options={
-        lines?.map(({ id, code }) => ({
+        lines?.map(({ id, name, code }) => ({
           id,
-          value: code,
+          value: programIndicatorCode === 'HIV' ? `${name}: ${code}` : code,
```

Comment:
There are no rules for Regimen indicator lines - my intuition is to put the actual question in the `name`, not the `code` so I can't see my questions still:

![image](https://github.com/user-attachments/assets/880301ad-986b-4754-bb88-87b5a57d2e97)

I firmly believe that we should show name and code regardless of the program_indicator. If we ever actually make this feature configurable properly like it was designed to be this will break (What happens when they want an extra program_indicator for TB programs?).